exec/Main.hs

```diff
@@ -89,6 +95,21 @@ main = do
       Args _ _ _ _ level _ -> level
       RichListArgs _ level -> level
 
+addTransactionsHeightIndex :: LogFunctionIO Text -> Connection -> IO ()
```

Comment:
We should probably make these functions check to see whether the index exists and skip the `create` if it does.